terraform/demo-api/main.tf

```diff
@@ -0,0 +1,59 @@
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_container_registry" "acr" {
+  name                = var.acr.name
+  resource_group_name = var.acr.resource_group_name
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = "rg-${var.env}-gitops-demo"
+  location = "Central US"
```

Comment:
```suggestion
  location = "westeurope"
```

Better latency

Reply:
Money. I'm using my personal account with a few euros for the first month 🤷